frontend/packages/ceph-storage-plugin/src/components/create-storage-system/create-storage-system-steps/security-and-network-step/encryption.tsx

```diff
@@ -204,9 +204,7 @@ export const Encryption: React.FC<EncryptionProps> = ({
   };
 
   const description = !isMCG
-    ? t(
-        'ceph-storage-plugin~Data encryption for block and file storage. MultiCloud Object Gateway is always encrypted.',
-      )
+    ? t('ceph-storage-plugin~Data encryption for block and file storage.')
```

Comment:
This is already migrated to odf-console, fix has to be in odf-console only. 

cc: @SanjalKatiyar

Reply:
yes @GowthamShanmugam u r right...

also, @TimothyAsirJeyasing plz so thr the bug comments before raising the PR https://bugzilla.redhat.com/show_bug.cgi?id=2038873#c10 
 I don't think we have done what is being asked...

Reply:
@SanjalKatiyar Do you mean we need a text like
ceph-storage-plugin~Data encryption for block and file storage. MultiCloud Object Gateway supports encryption for objects by default